### ISSUE TITLE
Add support for `SecArgumentSeparator` setting

### ISSUE THREAD

**Issue by fzipi** [corazawaf/coraza#1109]
## Summary

Do you want to engage with Coraza but don't know where to start? This is the perfect issue for you!

Let's add support for `SecArgumentSeparator` setting. This should be set in the `coraza.conf-recommended`, and the default is `&`.

👉 Where is this set? Easy, you can find the (hardcoded) value here (a): https://github.com/corazawaf/coraza/blob/6471debe70fd7795abf1ee935435ef3ec0477db8/internal/bodyprocessors/urlencoded.go#L26 

### I'm excited, this sounds awesome! What do I need to do?

1. Add a new option in the WAF in https://github.com/corazawaf/coraza/blob/c094576cdbf4523fe486b4ae7af35dc67aac68cd/internal/corazawaf/waf.go#L35. You can call it `ArgumentSeparator`, and it will be a char probably 😄 
2. Add support for the `SecArgumentSeparator` directive. This is easy: take a look at the [`internal/seclang/directives.go`](https://github.com/corazawaf/coraza/blob/main/internal/seclang/directives.go) file, and create a new `func directiveSecArgumentSeparator(options *DirectiveOptions) error`. It will be very similar to other functions, e.g. `directiveSecWebAppID`. You will need to store the value of the argument separator in the variable created in 1.
3. Just inject the variable's value instead of the hardcoded value shown above in (a)
4. Write a test proving that the directive works! Take a look for example at the tests [here](https://github.com/corazawaf/coraza/blob/c094576cdbf4523fe486b4ae7af35dc67aac68cd/internal/seclang/directives_test.go#L108) and add a simple test that proves the vale of the WAF option is changed when you update the value.
5. Create a PR! 🚀 

### Motivation

1. Bringing you to the awesome community we have in Coraza ❤️ 
2. Enjoy coding and learning Go and WAFs!
3. Fun. Because this is fun, isn't it? 😉

**Comment by jptosso** [comment]
I don't think we should implement this. In my whole career I have never seen the ; separator for url encoded

**Comment by fzipi** [comment]
No, there is no real use case. This was mostly about documenting the process. But I think the exercise was valuable and will add to the website docs and close.
